src/components/dashboard/content/list/FaultTreeAndSystemOverviewTable.tsx

```diff
@@ -5,11 +5,11 @@ import { useTranslation } from "react-i18next";
 import useStyles from "./FaultTreeOverviewTable.styles";
 import { FaultTree } from "@models/faultTreeModel";
 import { useNavigate } from "react-router-dom";
-import {ROUTES} from "@utils/constants";
+import { ROUTES } from "@utils/constants";
 import { extractFragment } from "@services/utils/uriIdentifierUtils";
 import { System } from "@models/systemModel";
-import { getModifiedSystemsList, getModifiedFaultTreesList, formatDate } from "@utils/utils";
-import {useSelectedSystem} from "@hooks/useSelectedSystem";
+import { filterFaultTreesBySystem, reorderSystemsList, formatDate } from "@utils/utils";
```

Comment:
I have convention to use get prefix if you do not modify the parameters of the function thus i would prefer:
`filterFaultTreesBySystem --> getFilteredFaultTreesBySystem`

Do you consider it reasonable or u feel conventions in javascript are different ?

Reply:
I understand the convention. getFilteredFaultTreesBySystem sounds reasonable